Makefile

```diff
@@ -388,6 +388,10 @@ test-gadgets: install/ig
 testdata:
 	$(MAKE) -C testdata/
 
+.PHONY: go-mod-tidy
+go-mod-tidy:
+	find ./ -type f -name go.mod -execdir go mod tidy \;
```

Comment:
LGTM but it will not work on my workflow ;)

While working on cilium/ebpf, I created a bind mount of cilium/ebpf in a subdirectory of the inspektor-gadget sources, and I used `replace github.com/cilium/ebpf => ./deps/ebpf`.

This allowed me to recompile ig fast with [depot.dev](https://depot.dev/) without having to push my cilium/ebpf on a remote git repository for every recompilation.

<details>

```patch
diff --git a/Dockerfiles/ig.Dockerfile b/Dockerfiles/ig.Dockerfile
index c264481d1..69b6ec2e4 100644
--- a/Dockerfiles/ig.Dockerfile
+++ b/Dockerfiles/ig.Dockerfile
@@ -13,7 +13,8 @@ ARG GOPROXY
 ENV GOPROXY=${GOPROXY}
 
 COPY go.mod go.sum /cache/
-RUN cd /cache && go mod download
+COPY deps /cache/deps
+RUN cd /cache && ls -l && go mod download
 ADD . /go/src/github.com/inspektor-gadget/inspektor-gadget
 
 WORKDIR /go/src/github.com/inspektor-gadget/inspektor-gadget
diff --git a/Dockerfiles/ig.Dockerfile.dockerignore b/Dockerfiles/ig.Dockerfile.dockerignore
index 1dc58b26b..94678bfb5 100644
--- a/Dockerfiles/ig.Dockerfile.dockerignore
+++ b/Dockerfiles/ig.Dockerfile.dockerignore
@@ -8,3 +8,4 @@
 !tools
 !Makefile*
 !*.mk
+!deps
diff --git a/go.mod b/go.mod
index f155a1034..8176e4c1f 100644
--- a/go.mod
+++ b/go.mod
@@ -224,7 +224,7 @@ require (
 
 //replace github.com/cilium/ebpf => github.com/cilium/ebpf v0.16.1-0.20241023154409-d3c63ab2edcb
 
-replace github.com/cilium/ebpf => /home/alban/go/src/github.com/cilium/ebpf
+replace github.com/cilium/ebpf => ./deps/ebpf
 
 replace k8s.io/component-base => k8s.io/component-base v0.0.0-20240417101527-62c04b35eff6
 
```

</details>

Reply:
Is there any need for the cilium dependency to be under the same Inspektor Gadget repo?

In any case, I think your setup is very specific, will merge the PR :D